spec/requests/jitsi_call_request_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Jitsi Call', type: :request do
+  describe 'POST' do
+    context 'when logged in' do
+      let(:participant) { FactoryBot.create(:user) }
+
+      before do
+        sign_in FactoryBot.create(:user)
+      end
+
+      it 'creates a new call' do
+        post jitsi_calls_path, params: { jitsi_call: { user_id: participant.id } }
+        expect(response).to have_http_status(:no_content)
+      end
+
+      it 'does not create a new call without participant user id' do
+        post jitsi_calls_path, params: { jitsi_call: { user_id: nil } }
+        expect(response).to redirect_to(root_path)
+      end
+    end
+
+    context 'without logging in' do
+      it 'redirects to login page' do
+        post jitsi_calls_path, params: { jitsi_call: { user_id: 0 } }
```

Comment:
I think you should use `participant.id` here too. Then it would be clear that the redirect is because of authentication and not because the not found logic maybe performs some weird redirect.

Reply:
Fixed this in 49cad28